README.md

```diff
@@ -1,11 +1,21 @@
 # KolibriOS-NG
 
-KolibriOS-NG is a fork of the [KolibriOS](https://kolibrios.org/) operating system.
+KolibriOS-NG is a fork of the [KolibriOS](https://kolibrios.org/) operating system
 
-KolibriOS is a hobby operating system for x86-compatible computers, its kernel is written entirely in [Fasm](https://flatassembler.net/) assembly language. Based on [MenuetOS](https://www.menuetos.net/), it uses its own standards and is not fully POSIX or UNIX compliant.
+## Overview
+
+KolibriOS is a hobby operating system for x86-compatible computers, its kernel is written entirely in [Fasm](https://flatassembler.net/) assembly language. Based on [MenuetOS](https://www.menuetos.net/), it uses its own standards and is not fully POSIX or UNIX compliant
+
+## Contributing
+
+A detailed description of the contribution process for those interested is described in the [contributing guide](./CONTRIBUTING.md)
```

Comment:
Might be a good idea to rephrase to avoid repetition, for instance:
```suggestion
Detailed instructions for the contribution process can be found in the [contributing guide](./CONTRIBUTING.md)
```